README.md

```diff
@@ -1,9 +1,21 @@
 # halfon
-Online AoEII Unit/Building/Tech reference for AoEII DE, AoEII HD, and AoEII WK
+Online AoE II Unit/Building/Tech reference for AoEII DE, AoEII HD, and AoEII WK.
 
-Age of Empires II © Microsoft Corporation.
-**Halfon** was created under Microsoft's "[Game Content Usage Rules](https://www.xbox.com/en-us/developers/rules)" using assets from Age of Empires II,
-and it is not endorsed by or affiliated with Microsoft.
+### Development Setup
+1. Checkout the code
+2. To run the website locally, either open `index.html` in a browser
```

Comment:
Just opening `index.html` in Firefox or Chrome will not work since they will not load the json files anymore.